corehq/apps/app_manager/suite_xml.py

```diff
@@ -1673,6 +1673,14 @@ def configure_entry_as_case_list_form(self, form, entry):
         frame_case_not_created.add_command(return_to)
         entry.stack.add_frame(frame_case_not_created)
 
+    def get_case_datums(self, module, form):
```

Comment:
Seems OK
On 22 Jul 2015 17:29, "Norman Hooper" <notifications@github.com> wrote:

> In corehq/apps/app_manager/suite_xml.py
> <https://github.com/dimagi/commcare-hq/pull/7586#discussion_r35225887>:
>
> > @@ -1673,6 +1673,14 @@ def configure_entry_as_case_list_form(self, form, entry):
> >          frame_case_not_created.add_command(return_to)
> >          entry.stack.add_frame(frame_case_not_created)
> >
> > +    def get_case_datums(self, module, form):
>
> What do you think of calling it get_case_datums_basic_module instead of a
> comment? A bit wordy?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/7586/files#r35225887>.
>